main.tf

```diff
@@ -0,0 +1,21 @@
+provider "aws" {
```

Comment:
Please remove this file from this pull request or move it to a new directory under `examples/` 👍

Reply:
whups, silly me :upside_down_face: